packages/labs/ssr-dom-shim/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@lit-labs/ssr-dom-shim",
+  "version": "1.0.0",
```

Comment:
Are we good with starting with 1.0.0? For most labs packages we've been starting with 0.1.0.

Reply:
I think we should just start with `1.0.0`, especially for Labs packages, because the "may change frequently" bit (which is what `0` major kinda means) is already implicit in the fact that this is a Labs package. And that allows us to make future updates with full major/minor/patch distinctions, instead of having to collapse minor/patch as we do when the major is 0.

Reply:
Oops, actually even given my explanation above, we do actually want to start from `0.0.1` or similar, because we'll want the first changeset entry to be a `major` bump, so that `1.0.0` is what actually gets published.